### PR TITLE
fix docs render

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Based on comment https://github.com/PyTorchLightning/pytorch-lightning/pull/5595#issuecomment-764706848 I have check our docs and found we have quite a lot of badly rendered docs :D

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by ydcjeff — APPROVED** [review]

**Review by s-rog — APPROVED** [review]
These seem pretty hard to catch without trawling through docs 😅

**Review by rohitgr7 — APPROVED** [review]

File: pytorch_lightning/plugins/ddp_sequential_plugin.py [hunk 1]
```diff
@@ -205,7 +206,9 @@ def _assert_valid_model_balance(self, trainer):
     def _skip_init_connections(self, trainer):
         """
         Skip initialization if torch is already initialized and we're in testing.
-        Returns: Whether to skip initialization
+
+        Returns:
+            Whether to skip initialization
```
Comment (akihironitta) [inline pytorch_lightning/plugins/ddp_sequential_plugin.py:212]: [nitpicking]
```suggestion
            Whether to skip initialization
```

File: pytorch_lightning/trainer/supporters.py [hunk 2]
```diff
@@ -382,7 +381,7 @@ def _wrap_loaders_max_size_cycle(self) -> Any:
         Wraps all loaders to make sure they are cycled until the longest loader is exhausted
 
         Returns:
-            Any: the wrapped loaders
+            the wrapped loaders
```
Comment (akihironitta) [inline pytorch_lightning/trainer/supporters.py:385]:
```suggestion
            the wrapped loaders
```

File: pytorch_lightning/trainer/supporters.py [hunk 3]
```diff
@@ -481,7 +480,7 @@ def __next__(self) -> Any:
         Fetches the next batch from multiple data loaders
 
         Returns:
-            Any: a collections of batch data
+            a collections of batch data
```
Comment (akihironitta) [inline pytorch_lightning/trainer/supporters.py:484]:
```suggestion
            a collections of batch data
```

File: pytorch_lightning/trainer/training_loop.py [hunk 4]
```diff
@@ -771,7 +771,8 @@ def block_ddp_sync_behaviour(self):
         do not block ddp gradient sync when using manual optimization
         as gradients are needed within the training step
 
-        Returns: context manager with sync behaviour off
+        Returns:
+            context manager with sync behaviour off
```
Comment (akihironitta) [inline pytorch_lightning/trainer/training_loop.py:776]:
```suggestion
            context manager with sync behaviour off
```

**Review by akihironitta — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
Did you find any tool to automate this ?

**Comment by Borda** [comment]
> Did you find any tool to automate this ?

not really, but it will be very welcome :]

**Review by williamFalcon — APPROVED** [review]

**Pull request merged**